### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by gabriel-cx**
**Proposed Changes**
- Docs updated information with the current status of the project

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by cxMiguelSilva — APPROVED** [review]
LGTM

File: docs/roadmap.md [hunk 1]
```diff
@@ -2,10 +2,15 @@
...
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+    - Kubernetes API
```
Comment (nunoocx) [inline docs/roadmap.md:11]: Adding K8S API in the set of technologies we support may be misleading about what it really is.
Let's put it as a feature separated from the techs support

**Review by nunoocx — CHANGES_REQUESTED** [review]

File: docs/roadmap.md [hunk 2]
```diff
@@ -2,10 +2,15 @@
...
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Kubernetes API
```
Comment (cxMiguelSilva) [inline docs/roadmap.md:11]: Maybe refer to Kubernetes API as kuberneter to reduce inconsistency between documentation
[docs/integrations_kuberneter.md](https://github.com/Checkmarx/kics/pull/5651/files#diff-d0564607bb5111b6ff222af74f8fc22b08d3537bb23c1878fad9562a62bcafb4)

**Review by cxMiguelSilva — CHANGES_REQUESTED** [review]

File: docs/roadmap.md (hunk 2 above)
Comment (gabriel-cx) [reply docs/roadmap.md:11]: which is the best approach in your opinion:
1) Kubernetes API ( Kuberneter )
2) Kuberneter Integration

**Review by cxMiguelSilva — DISMISSED** [review]
LGTM

File: docs/roadmap.md [hunk 3]
```diff
@@ -2,10 +2,15 @@
...
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Kuberneter (resource data gathering through Kubernetes API)
```
Comment (nunoocx) [inline docs/roadmap.md:11]: Maybe (dynamic gathering and scan of kubernetes cluster data through kubernetes API)

**Review by nunoocx — CHANGES_REQUESTED** [review]

File: docs/roadmap.md [hunk 4]
```diff
@@ -2,10 +2,15 @@
...
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Dynamic gathering and scan of Kubernetes cluster data through Kubernetes API
```
Comment (cxMiguelSilva) [inline docs/roadmap.md:11]:
```suggestion
- Kunerneter (dynamic gathering and scan of Kubernetes cluster data through Kubernetes API)
```
Shouldn't we keep 'Kuberneter'?
WDYT @nunoocx ?

Comment (gabriel-cx) [reply docs/roadmap.md:11]: I agree with that approach, having Kuberneter in the description

**Review by cxMiguelSilva — APPROVED** [review]
LGTM

File: docs/roadmap.md (hunk 4 above)
Comment (nunoocx) [reply docs/roadmap.md]: Yes I agree. My change proposal was just for the text within the parenthesis

File: docs/roadmap.md [hunk 5]
```diff
@@ -2,10 +2,15 @@
...
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Kuberneter (dynamic gathering and scan of Kubernetes cluster data through Kubernetes API
```
Comment (nunoocx) [inline docs/roadmap.md:11]:
```suggestion
- Kuberneter (dynamic gathering and scan of Kubernetes cluster data through Kubernetes API)
```

**Review by nunoocx — CHANGES_REQUESTED** [review]

File: docs/roadmap.md (hunk 4 above)
Comment (gabriel-cx) [reply docs/roadmap.md]: Done

**Review by nunoocx — APPROVED** [review]
LGTM

**Pull request merged**